flink-benchmark/pom.xml

```diff
@@ -57,6 +57,13 @@ under the License.
       <version>${jmh.version}</version>
       <scope>provided</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.flink</groupId>
+          <artifactId>flink-runtime</artifactId>
+          <version>0.10-SNAPSHOT</version>
```

Comment:
We don't hardcode the version for flink dependencies. Please use `<version>${project.version}</version>` instead.